node/runtime/src/lib.rs

```diff
@@ -452,26 +287,41 @@ impl_runtime_apis! {
 			Executive::finalise_block()
 		}
 
-		fn inherent_extrinsics(data: InherentData) -> Vec<UncheckedExtrinsic> {
-			data.create_inherent_extrinsics()
+		fn inherent_extrinsics(data: runtime_primitives::InherentData) -> Vec<<Block as BlockT>::Extrinsic> {
+			let mut inherent = Vec::new();
+
+			inherent.extend(
```

Comment:
manual amalgamation code?

Reply:
Yeah, I reverted in parts my macro here. I ran into some problems. As I will need to revisit the `InherentData` stuff anyway, I did not wanted to waste too much time on this topic in this pr.